src/WebUI/Components/KubeSummary.tsx

```diff
@@ -175,6 +187,15 @@ export class KubeSummary extends BaseComponent<IKubeSummaryProps, IKubernetesCon
                     key={format("dc-{0}", this.state.namespace || "")}
                     onItemInvoked={this._onDeploymentItemInvoked}
                 />
+                <DaemonSetListingComponent
+                    daemonSetList={this.state.daemonSetList || {} as V1DaemonSetList}
+                    key={format("daemonset-list-{0}", this.state.namespace||"all")}

```

Comment:
||"all" --> "" this is better, as there will be only one list, also make the key as smaller to make the html smaller